### ISSUE TITLE
[COGNITION]: Disability calculator needs required text added to the Disability Rating field

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#1039]
## Description

Our quick test logged an issue that we are not informing users the Disability Rating field is required. Screenshot attached below, but might need design consult for good layout.

/cc @smoonpeyes 

## Related Issues
* https://github.com/department-of-veterans-affairs/va.gov-team/issues/1045

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

* As a user, I want to know the Disability Rating field is required. I should be able to read, hear, and comprehend this through the use of text and color.

## Environment

* MacOS
* Chrome

## Possible Fixes (optional)
Adding a span for our required text block should fix this issue. It will need CSS to apply the correct color attributes.

```html
<div
  class="vads-l-col--4 small-screen:vads-l-col--3 vads-u-padding-right--2"
  id="ratingLabel"
>
  Disability rating
  <span
    class="schemaform-required-span"
    style="color: #cd2026;">
    (*Required)
  </span>
</div>
```

## WCAG or Vendor Guidance (optional)

* [Labels or Instructions: Understanding SC 3.3.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/minimize-error-cues.html)

## Screenshots or Trace Logs

<img width="778" alt="Screen Shot 2019-08-16 at 11 02 10 AM" src="https://user-images.githubusercontent.com/934879/63181325-58e8aa80-c015-11e9-8175-a1432b49763e.png">

**Comment by 1Copenut** [comment]
For this issue, @rtwell and I discussed it and we're going to take another tack:

1. Ryan will update the comp to have the second column label read "Description _(optional)_"
2. Can we add either `aria-required="true"` or `required` to the left column inputs? This will provide a cue to screen readers to announce the Disability rating is required.

**Comment by rtwell** [comment]
![image](https://user-images.githubusercontent.com/12564977/63363528-c7917500-c341-11e9-80ab-c6f5a58fef72.png)

**Comment by Karak888** [comment]
These have been captured moving to close.
